src/components/ShopList.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+    <div class="shops-list">
+        <article class="shop" v-for="shop in shops">
+           <img :src="shop.img" class="shop__img" />
+           <span class="shop__name">{{ shop.name }}</span>
+           <span class="shop__distance">{{ shop.distance }}</span>
+           <address class="shop__address">{{ shop.street }}</address>
+           <router-link :to='{path: `/shop-on-map/${shop.id}`}'>View on map</router-link>
+       </article>
+    </div>
+</template>
+
+<script>
+
+export default
+{
+    name: 'shop-list',
+    props: ['shops'],
```

Comment:
to do: get ot from the store in the future as other components that aren't children of /SearchShopByName.vue should have explicit access to this information. please mark it as required in the future implementation after ShopOnMap.vue component issues will be resolved.

Reply:
It goes through the same stream => io component, which manages all HTTP communication. If they're not required by other parts of the app they should not go to the store.

Reply:
OK. I am closing this issue as not significant to move on.